### PR TITLE
Swapped position of the - and + buttons on the shop page

### PR DESCRIPTION
**Pull request by JoseLRiano**
Screen Shot 2020-12-27 at 12 45 36 AM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,5 +58,20 @@
...
+    "type": "git",
+    "url": "git+https://github.com/JoseLRiano/website-crypto.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/JoseLRiano/website-crypto/issues"
+  },
+  "homepage": "https://github.com/JoseLRiano/website-crypto#readme"
 }
```
Comment (ankushdharkar) [inline package.json:77]: I don't think we need to update package.json for this change. Please remove these changes

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

**Review by ankushdharkar — APPROVED** [review]
Thank you for the changes @JoseLRiano! The changes will eventually show up on https://crypto.realdevsquad.com/shop

If you want to work more on such projects with us, please consider joining [our Discord](https://discord.gg/ge6NXX6wEQ)

**Comment by JoseLRiano** [comment]
Hi Ankush, I just remove those changes and push it to the branch I created.

**Comment by ankushdharkar** [comment]
Hello @JoseLRiano, 

I already accepted your PR. Thank you 🎉! It's merged into develop

**Pull request merged**